skewer.yaml

```diff
@@ -0,0 +1,117 @@
+title: Skupper Online Boutique
+subtitle: A Cloud-Native gRPC microservice-based application deployed across multiple Kubernetes clusters using Skupper
+overview: |
+  This tutorial demonstrates how to deploy the [Online
+  Boutique](https://github.com/GoogleCloudPlatform/microservices-demo/)
+  microservices demo application across multiple Kubernetes clusters that are
+  located in different public and private cloud providers. This project
+  contains a 10-tier microservices application developed by Google to
+  demonstrate the use of technologies like Kubernetes.
+
+  In this tutorial, you will create a Virtual Application Network that enables
+  communications across the public and private clusters. You will then deploy a
+  subset of the application's grpc based microservices to each cluster. You
+  will then access the `Online Boutique` web interface to browse items, add
+  them to the cart and purchase them.
+sites:
+  grpc-a:
+    title: gRPC A
+    platform: kubernetes
+    namespace: grpc-a
+    env:
+      KUBECONFIG: ~/.kube/config-grpc-a
+  grpc-b:
+    title: gRPC B
+    platform: kubernetes
+    namespace: grpc-b
+    env:
+      KUBECONFIG: ~/.kube/config-grpc-b
+  grpc-c:
+    title: gRPC C
+    platform: kubernetes
+    namespace: grpc-c
+    env:
+      KUBECONFIG: ~/.kube/config-grpc-c
+steps:
+  - standard: platform/access_your_kubernetes_clusters
+  - standard: platform/install_skupper_on_your_kubernetes_clusters
+  - title: Apply Kubernetes Resources
+    preamble: |
+      Apply the application deployment resources alongside the skupper
+      resources describing the application network.
+    commands:
+      grpc-a:
+        - run: kubectl create namespace grpc-a
```

Comment:
@ssorj this was awkward. I think in v1 skewer there was magic in a standard step to create namespaces based on the sites section.